src/resources/messages/index.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  PaginatedResponse,
+} from "../../common/interfaces";
+import {
+  Message,
+  MessageContent,
+  MessageEvent,
+} from "./interfaces";
+import {
+  Activity
+} from "../activities/interfaces";
+import { Knock } from "../../knock";
+
+export class Messages {
+  constructor(readonly knock: Knock) {}
+
+  async get(messageId: string): Promise<Message> {
+    if (!messageId) {
+      throw new Error(`Incomplete arguments. You must provide a 'messageId'`);
+    }
+
+    const { data } = await this.knock.get(`/v1/messages/${messageId}`);
+    return data;
+  }
+
+  async getContent(messageId: string): Promise<MessageContent> {
+    if (!messageId) {
+      throw new Error(`Incomplete arguments. You must provide a 'messageId'`);
+    }
+
+    const { data } = await this.knock.get(`/v1/messages/${messageId}/content`);
+    return data;
+  }
+
+  async getEvents(messageId: string): Promise<PaginatedResponse<MessageEvent>> {
```

Comment:
All of these paginated endpoints will need a second `options = {}` payload that can take the filters and the pagination params (`after`, `before`, `pageSize`) and will need to be passed as a query string